lib/structures/Message.js

```diff
@@ -65,36 +65,8 @@ class Message extends Base {
             this.member = null;
         }
         if(this.type === 0 || this.type === undefined);
-        else if(this.type === 1) {
-            data.content = `${this.author.mention} added <@${data.mentions[0].id}>.`;
-        } else if(this.type === 2) {
-            if(this.author.id === data.mentions[0].id) {
-                data.content = `@${this.author.username} left the group.`;
-            } else {
-                data.content = `${this.author.mention} removed @${data.mentions[0].username}.`;
-            }
-        } else if(this.type === 3) { // (╯°□°）╯︵ ┻━┻
-            if(data.call.ended_timestamp) {
-                if((!this.channel.lastCall || this.channel.lastCall.endedTimestamp < Date.parse(data.call.ended_timestamp))) {
-                    data.call.id = this.id;
-                    this.channel.lastCall = new Call(data.call, this.channel);
-                }
-                if(data.call.participants.includes(client.user.id)) {
-                    data.content = `You missed a call from ${this.author.mention}.`;
-                } else {
-                    data.content = `${this.author.mention} started a call.`;
-                }
-            } else {
-                if(!this.channel.call) {
-                    data.call.id = this.id;
-                    this.channel.call = new Call(data.call, this.channel);
-                }
-                data.content = `${this.author.mention} started a call. — Join the call.`;
-            }
-        } else if(this.type === 4) {
-            data.content = `${this.author.mention} changed the channel name: ${data.content}`;
-        } else if(this.type === 5) {
-            data.content = `${this.author.mention} changed the channel icon.`;
+        else if(this.type === 1 || this.type === 2 || this.type === 3 || this.type === 4 || this.type === 5 ) { // (╯°□°）╯︵ ┻━┻
```

Comment:
this isn't really worthy of a table flip lol

Reply:
Wait, don't do this check from within the constructor. Remove instances of the constructor being called with these types in the first place.

Reply:
Will change this tomorrow